crypto/hash.go

```diff
@@ -0,0 +1,70 @@
+package crypto
+
+import (
+	"encoding/hex"
+	"golang.org/x/crypto/blake2b"
+	"golang.org/x/crypto/ripemd160"
+)
+
+func Blake256(data string) string {
+	b, _ := blake2b.New(32, nil)
+	b.Write([]byte(data))
+	bs := b.Sum(nil)
+	var result []byte = make([]byte, 64)
+	hex.Encode(result, bs[:])
+	return string(result)
+}
+
+func Blake256WithKey(data string, key string) string {
```

Comment:
The type key in `codechain-primitive-js` is `Uint8Array` not string.

Reply:
I used []byte because space of Uint8 is identical to `[]byte`, 
And also blake2b golang implementation takes key as `[]byte`

Reply:
I agree with you.